core/input/keysym.js

```diff
@@ -94,6 +94,7 @@ export default {
     XK_KP_Next:                     0xff9b,
     XK_KP_Page_Down:                0xff9b,
     XK_KP_End:                      0xff9c,
+    XK_KP_Clear:                    0xff9d,
```

Comment:
However this is wrong. There is no such keysym. Use `XK_KP_Begin`, which is what a Unix system generates for that key.